packages/seedless-onboarding-controller/src/SeedlessOnboardingController.test.ts

```diff
@@ -2344,6 +2188,65 @@ describe('SeedlessOnboardingController', () => {
         },
       );
     });
+
+    it('should throw an error if vault unlocked has invalid authentication data', async () => {
+      const mockVault = JSON.stringify({ foo: 'bar' });
+
+      await withController(
+        {
+          state: getMockInitialControllerState({
+            withMockAuthenticatedUser: true,
+            vault: mockVault,
+          }),
+        },
+        async ({ controller, encryptor }) => {
+          jest
+            .spyOn(encryptor, 'decryptWithKey')
+            .mockResolvedValueOnce(mockVault);
+          await expect(
+            controller.submitPassword(MOCK_PASSWORD),
+          ).rejects.toThrow(
+            SeedlessOnboardingControllerErrorMessage.InvalidVaultData,
```

Comment:
Are you throwing a different error here than before? If so, this can be considered a breaking change (depending on your definition of breaking change).